cypress/integration/home_page_spec.js

```diff
@@ -0,0 +1,48 @@
+describe('Not Fur-gotten Home Page User Flow', () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        method: 'POST',
+        url: 'https://not-fur-gotten-be.herokuapp.com/graphql',
+        headers: {
+          'x-gql-operation-name': 'getAllPets',
```

Comment:
A lot of work finding the solution to interception a query.  discovering that we needed to use more items from our middleware Apollo Client.